tests/unit/test_charm.py

```diff
@@ -58,18 +57,6 @@ def test_given_prometheus_relation_not_created_when_pebble_ready_then_charm_goes
             "Waiting for prometheus relation to be created"
         )
 
-    @patch("charm.AlertRulesDirWatcher", Mock())
```

Comment:
Why do we remove this test?

Reply:
New ops.testing, on `container_pebble_ready` automatically sets `can_connect` to True, which makes this test useless.